apps/devhub/tests/test_views.py

```diff
@@ -3001,10 +2934,8 @@ def test_listed_beta_pass_validation_with_flag(self, mock_sign_file):
         assert log.action == amo.LOG.BETA_SIGNED_VALIDATION_PASSED.id
 
     @mock.patch('devhub.views.sign_file')
-    def test_listed_beta_do_not_pass_validation_with_flag(self,
-                                                          mock_sign_file):
+    def test_listed_beta_do_not_pass_validation(self, mock_sign_file):
         """Beta files that don't pass validation should raise an error."""
```

Comment:
Not related to this PR, but just realized the docstring is wrong: it shouldn't raise an error, it should simply log that it failed validation. Do you think you could fix that at the same time?